source/support.tex

```diff
@@ -3891,12 +3881,30 @@
   template<class... Ts>
     using common_comparison_category_t = typename common_comparison_category<Ts...>::type;
 
+  // \ref{cmp.concept}, concept \tcode{ThreeWayComparable}
+  template<class T, class Cat = partial_ordering>
+    concept ThreeWayComparable = see below;
```

Comment:
Shouldn't this use \seebelow or whatnot?

Reply:
Fixed.